pilot/lib/pilot/build_manager.rb

```diff
@@ -366,6 +366,9 @@ def expire_previous_builds(build)
     # If there are multiple teams, infer the provider from the selected team name.
     # If there are fewer than two teams, don't infer the provider.
     def transporter_for_selected_team(options)
+      # Ensure that user is authenticated
+      start(options)
```

Comment:
Hi @joshdholtz 

I think, this will not perform login and **will early exit** in this line [see here](https://github.com/fastlane/fastlane/blob/master/pilot/lib/pilot/manager.rb#L14)

because
`@ config` is already set by the `upload` method without login, [see here ](https://github.com/fastlane/fastlane/blob/master/pilot/lib/pilot/build_manager.rb#L17)